test/models/location.js

```diff
@@ -8,13 +8,8 @@ const ViewModel = require('../../server/models/views/location')
 const data = require('../data')
 
 lab.experiment('Outlook model test', () => {
-  let sandbox
-
-  lab.beforeEach(async () => {
-    sandbox = await sinon.createSandbox()
-  })
   lab.afterEach(async () => {
-    await sandbox.restore()
+    await sinon.restore()
```

Comment:
There is no mocking in this test so this line and the `require('sinon')` can be removed. I should have spotted this in my last review!